alf/bin/verify_checkpoint.py

```diff
@@ -0,0 +1,211 @@
+# Copyright (c) 2021 Horizon Robotics and ALF Contributors. All Rights Reserved.
```

Comment:
Should we add this to CI for all checked in algorithms?

Reply:
I thought about this. Then decided to do it later until this is more mature.